deploy.sh

```diff
@@ -135,16 +135,16 @@ svn cp "trunk" "tags/$VERSION"
 # Fix screenshots getting force downloaded when clicking them
 # https://developer.wordpress.org/plugins/wordpress-org/plugin-assets/
 if test -d "$SVN_DIR/assets" && test -n "$(find "$SVN_DIR/assets" -maxdepth 1 -name "*.png" -print -quit)"; then
-    svn propset svn:mime-type "image/png" "$SVN_DIR/assets/*.png" || true
+    svn propset svn:mime-type "image/png" "$SVN_DIR/assets/"*.png || true
```

Comment:
I think we can remove the quote entirely instead of partial quoting the file path glob.
```suggestion
    svn propset svn:mime-type "image/png" $SVN_DIR/assets/*.png || true
```

Reply:
Same for other file types.

Reply:
I think it's best to leave it quoted as the variable `$SVN_DIR` is not guaranteed to never have spaces in its value. If GitHub change their setup, then the `$HOME` directory might change to a location that includes a space. Therefore defensive programming suggests that we should retain the quotes to mitigate potential future breakage caused by platform changes.

Reply:
Make sense to me! Thanks so much for explanation!